yocto/docker/run-docker.sh

```diff
@@ -13,11 +13,20 @@ if [ "$2" ];then
 	EXTRA_ARGS="--volume $2:/tmp/sshagent --env=SSH_AUTH_SOCK=/tmp/sshagent"
 fi
 
+if [ ! "$3" ]; then
+	echo "Wrong usage: provide mirror build directory as parameter"
+	exit
+fi
+
+
 docker run \
  -it \
 ${EXTRA_ARGS} \
+--mount type=bind,source=/var/run/pcscd,target=/var/run/pcscd \
```

Comment:
this should also be an optional argument

Reply:
is optional now